docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -51,7 +49,7 @@ To prepare credentials for a local Teleport user:
 
 1. Create a new file called `terraform.yaml` and open it in an editor.
 
-1. Configure settings for a local Teleport user and role by pasting the following content into the `terraform.yaml` file:
+1. Configure the necessary role by pasting the following content into a `terraform_role.yaml` file:
```

Comment:
I would clarify what entity will have this role and what it uses the role for

Reply:
Done, let me know what you think of the new language.